inorbit_republisher/scripts/republisher.py

```diff
@@ -49,9 +49,16 @@ def main():
     # Start the ROS node
     rospy.init_node('inorbit_republisher', anonymous=True)
 
-    # Read republisher configuration from the 'config' parameter
+    # Read republisher configuration from the 'config' or 'config_file' parameter
```

Comment:
```suggestion
    # Read republisher configuration from the 'config_file' or 'config' parameter
```

Reply:
Changing the order to reflect the way this is prioritizing file over mere config if both are present.